e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -26,6 +29,7 @@ describe('Customization', () => {
     });
 
     after(() => {
+        origConfig.TeamSettings.EnableCustomBrand = false;
```

Comment:
Instead of this, you may do https://github.com/mattermost/mattermost-webapp/pull/4391#discussion_r353651197.